nilearn/_utils/numpy_conversions.py

```diff
@@ -124,3 +126,46 @@ def as_ndarray(arr, copy=False, dtype=None, order='K'):
         raise ValueError("Type not handled: %s" % arr.__class__)
 
     return ret
+
+
+def csv_to_array(csv_path, delimiters=' \t,;', **kwargs):
+    """Read a CSV file by trying to guess its delimiter
+
+    Parameters
+    ----------
+
+    csv_path: string
+        Path of the CSV file to load.
+
+    delimiters: string
+        String containing the possible delimiters for the CSV file.
+
+    **kwargs: keyword arguments
+        The additional keyword arguments are passed to numpy.genfromtxt when
+        loading the CSV.
+
+    Returns
+    -------
+
+    array: numpy.ndarray
+        An array containing the data loaded from the CSV file.
+    """
+    if not isinstance(csv_path, _basestring):
+        raise TypeError('CSV must be a file path. Got a CSV of type: %s' %
+                        type(csv_path))
+    # First, we try genfromtxt which works in most cases.
+    array = np.genfromtxt(csv_path, **kwargs)
```

Comment:
Would it not make sense to extend the SEH frame (except, etc.) to ``np.genfromtxt()``?